draft-ietf-quic-transport.md

```diff
@@ -2678,72 +2678,105 @@ transmission efficiency to underfilled packets.
+  until acknowledged or until acknowledgments for all stream data is received
+  from the peer (that is, either the "Reset Recvd" or "Data Recvd" state is
+  reached on the send stream).  The content of a RST_STREAM frame MUST NOT
+  change when it is sent again.
 
-* ACK and PADDING frames MUST NOT be retransmitted.  ACK frames
-  containing updated information will be sent as described in {{frame-ack}}.
+* Similarly, requests to cancel stream transmission, as encoded in a
+  STOP_SENDING frame, is sent until the receive stream enters either a "Data
+  Recvd" or "Reset Recvd" state, see {{solicited-state-transitions}}.
 
-* STOP_SENDING frames MUST be retransmitted until the receive stream enters
-  either a "Data Recvd" or "Reset Recvd" state.  See
-  {{solicited-state-transitions}}.
+* Connection close signals, including those that use CONNECTION_CLOSE and
+  APPLICATION_CLOSE frames, are not sent again when packet loss is detected, but
+  as described in {{termination}}.
 
-* The most recent MAX_STREAM_DATA frame for a stream MUST be retransmitted until
-  the receive stream enters a "Size Known" state. Any previous unacknowledged
-  MAX_STREAM_DATA frame for the same stream SHOULD NOT be retransmitted since a
-  newer MAX_STREAM_DATA frame for a stream obviates the need for delivering
-  older ones. Similarly, the most recent MAX_DATA frame MUST be retransmitted;
-  previous unacknowledged ones SHOULD NOT be retransmitted.
+* The current maximum data offset is sent in MAX_DATA frames until acknowledged.
+  Note that increases to the offset might separately cause new MAX_DATA frames
+  to be sent.  Care is necessary to limit the number of times this frame is
+  sent.
 
-* BLOCKED, STREAM_BLOCKED, and STREAM_ID_BLOCKED frames SHOULD be retransmitted
-  if the sender is still blocked on the same limit.  If the limit has been
-  increased since the frame was originally sent, the frame SHOULD NOT be
-  retransmitted.
+* The current maximum stream data offset is sent in MAX_STREAM_DATA frames until
+  acknowledged or the receive stream enters a "Size Known" state.  Like
+  MAX_DATA, endpoints need to limit the rate at which this frame is sent.
 
-* All other frames MUST be retransmitted.
+* The maximum stream ID is sent in MAX_STREAM_ID frames until acknowledged.
+  Like MAX_DATA and MAX_STREAM_DATA, endpoints might need to limit the rate at
+  which this frame is sent.
+
+* Blocked signals are carried in BLOCKED, STREAM_BLOCKED, and STREAM_ID_BLOCKED
+  frames until acknowledged, but only while the endpoint is blocked on the
+  corresponding limit.  These frames always include the limit that is causing
+  blocking at the time that they are transmitted.
+
+* An liveness or path validation check using PING frames is sent until
+  acknowledged or until there is no remaining need for liveness or path
+  validation checking.  PING frames with a payload SHOULD include a different
+  payload each time they are sent.
```

Comment:
PING frames don't have a payload any more.

Reply:
And related, the merge conflict is probably that this section doesn't mention PATH_CHALLENGE / PATH_RESPONSE frames.

Reply:
You will note that date at which this PR was created relative to when it is being reviewed.  I have rebased.